src/components/content-tab-sync.tsx

```diff
@@ -21,18 +20,17 @@ function SiteSyncDescription( { children }: PropsWithChildren ) {
 		<div className="flex justify-between max-w-3xl gap-4">
 			<div className="flex flex-col p-8">
 				<div className="flex items-center mb-1">
-					<div className="a8c-subtitle">{ __( 'Sync with' ) }</div>
-					<WordPressShortLogo className="ms-2 h-5" />
+					<div className="a8c-subtitle">{ __( 'Sync with WP.com or Pressable' ) }</div>
 				</div>
 				<div className="max-w-[40ch] text-a8c-gray-70 a8c-body">
 					{ __(
-						'Connect your existing WordPress.com or Jetpack-activated Pressable sites, or create a new one. Then, share your work with the world.'
+						'Connect your existing WordPress.com or Pressable sites with Jetpack activated, or create a new one. Then share your work with the world.'
```

Comment:
Nit: In this sentence, I think the "with Jetpack activated" part is a bit ambiguous. It's a bit unclear to me whether Jetpack applies to both WordPress.com and Pressable or just Pressable.

What do you think about changing it slightly to make it more clear? eg.

```suggestion
						'Connect your existing WordPress.com site, a Pressable site with Jetpack activated, or create a new one.'
```

Reply:
Thank you for your review, Gergely. I see your point and I also like the use of singular for site - as even the button below says `Connect site`:

![Markup on 2025-05-16 at 15:22:59](https://github.com/user-attachments/assets/700d005f-37b4-4f55-981b-c422a4811843)

What do you think, @sejas?